### PR TITLE
Add issues listing for imports

### PR DESCRIPTION
**Pull request by wejendorp**
New UI for issues from local-cluster-service, should help resolve issues w/ invalid or missing assets due to invalid yml files.

Screenshot 2024-04-04 at 14 18 08
Screenshot 2024-04-04 at 14 18 16

Screenshot 2024-04-04 at 09 50 25

Screenshot 2024-04-04 at 14 18 51

## Requires 
- [x] https://github.com/kapetacom/local-cluster-service/pull/134
- [x] https://github.com/kapetacom/local-cluster-config/pull/13

### CODE REVIEW

**Comment by hofmeister** [comment]
Haven't reviewed yet - just noticed the typo "Unknown asset as" - should probably be "Unknown asset **at**"

File: src/renderer/components/shell/TopBar.tsx [hunk 1]
```diff
@@ -75,6 +85,20 @@ export const TopBar = withTheme((props: TopBarProps) => {
                     height="100%"
                     alignItems="center"
                 >
+                    {props.issues?.length ? (
```
Comment (hofmeister) [inline src/renderer/components/shell/TopBar.tsx:88]: Typically we try to disable - not hide - buttons when not in effect. Just a better UX principle in terms of learning people where things are

Comment (hofmeister) [reply src/renderer/components/shell/TopBar.tsx:88]: And if disabled - ideally some tooltip to explain why its disabled (e.g. "No issues currently found)

Comment (hofmeister) [reply src/renderer/components/shell/TopBar.tsx:88]: Alternative would be to always enable - add a number "badge" to it - and make it green and zero when there are no issues. And then handle zero in the modal as well

**Review by hofmeister — CHANGES_REQUESTED** [review]

File: src/renderer/components/shell/TopBar.tsx (hunk 1 above)
Comment (wejendorp) [reply src/renderer/components/shell/TopBar.tsx:88]: Ah yea makes sense, I forgot about that, thanks.

**Comment by wejendorp** [comment]
Changed button to always active, and switched the icon to a tick when everything is ok.

**Review by hofmeister — APPROVED** [review]

**Pull request merged**